ui/src/app/pages/workspace/workspace-wrapper.tsx

```diff
@@ -86,25 +98,38 @@ const NewCtNotification = (props: NotificationProps) => {
   );
 
   return (
-    <div style={styles.newCtNotification}>
-      <FlexRow style={{ justifyContent: 'space-between' }}>
-        <FlexColumn>
-          <div>
-            These resources will get you started using the genomic data:{' '}
-            {dataPathsLink}, {dataFilesLink}, and {cohortBuilderLink}.
-          </div>
-          <div>
-            You can always learn more in the {supportHubLink} section on
-            Genomics.
-          </div>
-        </FlexColumn>
-        <FontAwesomeIcon
-          icon={faXmark}
-          style={{ fontSize: '21px', marginLeft: '1em' }}
-          onClick={() => props.onCancel()}
-        />
-      </FlexRow>
-    </div>
+    <BannerNotification>
+      <FlexColumn>
+        <div>
+          These resources will get you started using the genomic data:{' '}
+          {dataPathsLink}, {dataFilesLink}, and {cohortBuilderLink}.
+        </div>
+        <div>
+          You can always learn more in the {supportHubLink} section on Genomics.
+        </div>
+      </FlexColumn>
+      <FontAwesomeIcon
+        icon={faXmark}
+        style={{ fontSize: '21px', marginLeft: '1em' }}
+        onClick={() => props.onCancel()}
+      />
+    </BannerNotification>
```

Comment:
This is an indentation change after refactoring the outermost `div` and `FlexRow` into `BannerNotification`.